app/views/planning_application/consultations/new.html.erb

```diff
@@ -14,24 +14,16 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full" data-controller="address-fill polygon-search">
-    <%= render(
-      partial: "shared/location_map",
-      locals: {
-        locals: {
-          in_accordion: false,
-          polygon_search: true,
-          geojson: @planning_application.boundary_geojson,
-          geojson_field: "polygon-geojson-field"
-        }
-      }
-    ) %>
+    <%= render "select_neighbours", geojson: @planning_application.boundary_geojson, consultation: @consultation %>
 
     <%= render "neighbour_addresses_form", planning_application: @planning_application, consultation: @consultation %>
 
-    <%= render "neighbours_added", planning_application: @planning_application, consultation: @consultation %>
-    
+    <p class="govuk-body">or</p>
```

Comment:
Is this meant to say just "or" on its own? It's not very clear in template form (although it might make sense in the context of the partials around it).

Reply:
Yeah it's just an "or" 😄